src/components/forms/SubnetTileForm.js

```diff
@@ -65,21 +65,51 @@ class SubnetTileForm extends React.Component {
           className="marginBottomSmall"
         />
         <div className="displayFlex">
-          {subnetMap.map((subnet) => (
-            <SubnetForm // change so doesn't show buttons
-              key={`${subnet.name}-tile-${this.props.tier}-${
-                this.props.vpc_name
-              }-${JSON.stringify(subnet)}`}
-              vpc_name={this.props.vpc_name}
-              data={subnet}
-              onSave={this.props.onSave}
-              isModal={this.props.isModal || this.props.readOnly}
-              componentDidUpdateCallback={this.childSubnetHasChanged}
-              networkAcls={this.props.networkAcls}
-              disableSaveCallback={this.props.disableSaveCallback}
-              shouldDisableGatewayToggle={this.shouldDisableGatewayToggle}
-            />
-          ))}
+          {subnetMap.map((subnet, index) => {
+            if (
+              this.props.advanced &&
+              !contains(this.props.select_zones, index + 1)
+            ) {
+              return (
+                <SubnetForm // change so doesn't show buttons
```

Comment:
is this comment a later thing or still a todo for this issue

Reply:
this is an old comment will delete